spec/unit/lib/cloud_controller/diego/app_recipe_builder_spec.rb

```diff
@@ -216,7 +221,8 @@ module Diego
               global_environment_variables: env_vars,
               privileged?:                  false,
               ports:                        lrp_builder_ports,
-              port_environment_variables:   expected_action_environment_variables,
+              port_environment_variables:   expected_action_port_variables,
```

Comment:
Could we revert all changes to `app_recipe_builder_spec`, this change should not have impacted the `AppRecipeBuilder` in anyway? I'll watch this PR and merge it when that happens, so you don't have to wait for the next community pair to pick this up.